hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/SerializationTest.java

```diff
@@ -264,6 +278,14 @@ void dualReadAndWrite() throws IOException, ConstructableRegistryException {
                 .reset();
 
         assertTree(loadedTreeWithCache);
+        try {
+            originalTree.release();
+        } catch (Exception e) {
+            logger.error("Exception while releasing state", e);
+        }
```

Comment:
@imalygin, I think it's best to remove this altogether, since, according to my observation, it always throws an exception.